src/index.html

```diff
@@ -10,13 +10,102 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+          class="nav__logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list-items">
+          <li class="nav__list-item">
+            <a
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Instead of `nav__list-link` and `nav__list-item`, consider more semantic names that describe the role of the element in the navigation, such as `nav__link` and `nav__item`.